ci-operator/jobs/devfile/integration-tests/devfile-integration-tests-main-periodics.yaml

```diff
@@ -12,10 +12,10 @@ periodics:
   labels:
     ci-operator.openshift.io/cloud: aws
     ci-operator.openshift.io/cloud-cluster-profile: aws-devfile
-    ci-operator.openshift.io/variant: v4.10.console
+    ci-operator.openshift.io/variant: v4.11.console
```

Comment:
v4.11?

Reply:
@johnmcollier Changes to the jobs are reordered so its changing all of them to the next one up, same result though.